src/NexusRpc/Handlers/OperationStartResult.cs

```diff
@@ -0,0 +1,64 @@
+#pragma warning disable SA1402 // We allow multiple types of the same name
+
+namespace NexusRpc.Handlers
+{
+    /// <summary>
+    /// Helper for results of an operation start.
+    /// </summary>
+    public static class OperationStartResult
+    {
+        /// <summary>
+        /// Create sync result with fixed result value.
+        /// </summary>
+        /// <typeparam name="TResult">Result type.</typeparam>
+        /// <param name="resultValue">Synchronous result value.</param>
+        /// <returns>Start result with the result value.</returns>
+        public static OperationStartResult<TResult> SyncResult<TResult>(TResult resultValue) =>
```

Comment:
Can we call these factories `Sync` and `Async`? Is there a .NET reason to stutter here?